processes/github_access.md

```diff
@@ -20,8 +20,13 @@ All projects at Defra must be created under one of the organisations within the
 - [Defra](https://github.com/DEFRA) for software
 - [Defra Data Science Centre of Excellence](https://github.com/-Defra-Data-Science-Centre-of-Excellence) for data science
 - [Defra design team](https://github.com/defra-design) for prototype designs
+- [aphascience](https://github.com/aphascience) for the APHA Scientific Computing Environment
```

Comment:
Technically the APHAScience org isn't just for SCE activity, though it's mostly used for SCE related work.  Could say `for scientific projects at APHA`.

Reply:
Changed